etc/kubernetes/helm/enterprise-gateway/templates/service.yaml

```diff
@@ -7,21 +8,26 @@ metadata:
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+  {{- range $key, $val := .Values.global.commonLabels }}
+    {{ $key }}: "{{ $val }}"
+  {{- end }}
+
   name: enterprise-gateway
   namespace: {{ .Release.Namespace }}
 spec:
   ports:
-  - name: http
-    port: {{ .Values.port }}
-    targetPort: {{ .Values.port }}
-  - name: response
-    port: {{ .Values.response_port }}
-    targetPort: {{ .Values.response_port }}
+    {{- range $key, $val := .Values.service.ports }}
+    -
+      {{- range $pkey, $pval := $val }}
+      {{ $pkey}}: {{ $pval }}
+      {{- end }}
+    {{- end }}
   selector:
     gateway-selector: enterprise-gateway
   sessionAffinity: ClientIP
-  type: NodePort
-{{- if .Values.k8sMasterPublicIP }}
+  type: {{ .Values.service.type }}
+  {{- if .Values.service.externalIPs.k8sMasterPublicIP }}
   externalIPs:
-  - {{ .Values.k8sMasterPublicIP }}
+  - {{ .Values.sertvice.externalIPs.k8sMasterPublicIP }}
```

Comment:
typo in service, `sertvice` -> `service`

Reply:
Thanks @bloomsa - great catch.  I'll push the update since I have the latest.  I see a couple of other typos in the docs as well and will try to fix all I can.

Looks like the `tbump` issue is resolved, so I think we're close to merging this! The link check will fail until merged.